### PR TITLE
Update for recent container changes

### PR DESCRIPTION
**Pull request by spilchen**

### CODE REVIEW

File: content/containers/creating-image.md [hunk 1]
```diff
@@ -309,6 +305,26 @@ As previously mentioned, the second stage uses Ubuntu as the operating system. Y
...
+The third and final build stage:
+
+- copies only the necessary build artifacts from the first two stages to reduce the number of layers in the final image.
+- creates environment variables.
+- exposes ports for networking.
+- adds metadata to the image.
+
+#### Set Up Stage
+
+Start with an empty image called scratch and copy everything over from the second stage:
```
Comment (rs-vertica) [inline content/containers/creating-image.md:319]:
```suggestion
Start with an empty image called `scratch` and copy everything from the second stage:
```

File: content/containers/creating-image.md [hunk 2]
```diff
@@ -309,6 +305,26 @@ As previously mentioned, the second stage uses Ubuntu as the operating system. Y
...
+
+```
+FROM scratch
+COPY --from=initial / /
+```
+
+This gets rid of any intermediate layer that was created to update the OS packages.
```
Comment (rs-vertica) [inline content/containers/creating-image.md:326]:
```suggestion
This removes any intermediate layers that were created to update the OS packages.
```

File: content/containers/creating-image.md [hunk 3]
```diff
@@ -455,11 +470,23 @@ Add the following only if you are building a minimal image:
   fi \
 ```
 
+##### Install vim
+
+Add the following for debugging purposes. The vim package can be quite old. Some security scanners may find vulnerabilities in it. For this reason, we don't include it in the NO_KEYS image, which should be used in environments with strict security requirements.
```
Comment (rs-vertica) [inline content/containers/creating-image.md:475]:
```suggestion
Add the Vim text editor for debugging purposes. The `vim` package can be old, and some security scanners might find vulnerabilities in it. For this reason, it is not included it in the NO_KEYS image, which should be used in environments with strict security requirements.
```

File: content/containers/creating-image.md [hunk 4]
```diff
@@ -544,11 +571,12 @@ ENTRYPOINT ["/init"]
 
 ### Exposing Ports
 
-Expose port 5433 for Vertica, and 8443 for Vertica's HTTP server:
+Expose port 5433 for Vertica, 8443 for Vertica's HTTP server and 5444 for vertic agent:
```
Comment (rs-vertica) [inline content/containers/creating-image.md:574]:
```suggestion
Expose port 5433 for Vertica, 8443 for Vertica's HTTP server, and 5444 for vertic agent:
```

File: content/containers/creating-image.md [hunk 5]
```diff
@@ -292,12 +293,7 @@ Ensure that everything under /home/dbadmin has the correct ownership and the ssh
 
 ### Second Stage
 
-The second and final build stage:
-
-- copies only the necessary build artifacts from the first stage to   reduce the number of layers in the final image.
-- creates environment variables.
-- exposes ports for networking.
-- adds metadata to the image.
+The second stage sets up the OS to be used for the final image. It updates the packages so that we are using the latest ones with all known security vulnerabilities addressed. This stage hands off to the third and final stage to remove any intermediate files needed for the package update.
```
Comment (rs-vertica) [inline content/containers/creating-image.md:296]:
```suggestion
The second stage prepares the OS for the final image. It updates packages to address all known security vulnerabilities. This stage hands off to the third and final stage to remove any intermediate files that are needed for the package update.
```

**Review by rs-vertica — CHANGES_REQUESTED** [review]
changes look great!

**Comment by spilchen** [comment]
Thanks for reviewing Ryan. I accepted all of your suggestions.

**Review by rs-vertica — APPROVED** [review]

**Pull request merged**